science/rtl_433/Portfile

```diff
@@ -5,26 +5,33 @@ PortGroup           cmake 1.1
 PortGroup           github 1.0
 
 epoch               1
-github.setup        merbanan rtl_433 21.12
-# Change github.tarball_from to 'releases' or 'archive' next update
-github.tarball_from tarball
+github.setup        merbanan rtl_433 25.02
+github.tarball_from archive
 revision            0
 
 categories          science comms
 license             GPL-2
-maintainers         {@ducksauz duksta.org:john} openmaintainer
+maintainers         nomaintainer
 
 description         RTL-SDR 433.92 MHz generic data receiver
-long_description    rtl_433 turns your Realtek RTL2832 based DVB dongle into a 433.92 MHz generic data receiver
+long_description    rtl_433 turns your Realtek RTL2832 based DVB dongle \
+                    into a 433.92 MHz generic data receiver
 
-checksums           rmd160  2610e3882d83877c291f9a2cef421e03470135b1 \
-                    sha256  8bc1866f61f8c2b4f6140c7b9ce159187e5ab821ecfba1ae2e35cde34423b8b9 \
-                    size    954737
+checksums           rmd160  255a8154784c2c46aafa2a500e0ceb560ed9c7b1 \
+                    sha256  5a409ea10e6d3d7d4aa5ea91d2d6cc92ebb2d730eb229c7b37ade65458223432 \
+                    size    1125144
 
-depends_build-append \
-                    port:pkgconfig
+cmake.out_of_source yes
```

Comment:
this is the default set by the `cmake` PortGroup and should thus be removed

Reply:
Done.